src/modules/previewpane/UnitTests-SvgPreviewHandler/SvgPreviewControlTests.cs

```diff
@@ -93,61 +94,16 @@ public void SvgPreviewControl_ShouldSetScrollBarsEnabledProperty_WhenDoPreviewCa
             Assert.AreEqual(((WebBrowser)svgPreviewControl.Controls[0]).ScrollBarsEnabled, true);
         }
 
-        [TestMethod]
-        public void SvgPreviewControl_ShouldAddTextBox_IfBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));
-
-            // Assert
-            Assert.IsInstanceOfType(svgPreviewControl.Controls[0], typeof(RichTextBox));
-            Assert.IsInstanceOfType(svgPreviewControl.Controls[1], typeof(WebBrowser));
-            Assert.AreEqual(svgPreviewControl.Controls.Count, 2);
-        }
-
-        [TestMethod]
-        public void SvgPreviewControl_ShouldNotAddTextBox_IfNoBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>valid</svg>"));
-
-            // Assert
-            Assert.IsNotInstanceOfType(svgPreviewControl.Controls[0], typeof(RichTextBox));
-            Assert.AreEqual(svgPreviewControl.Controls.Count, 1);
-        }
-
-        [TestMethod]
-        public void SvgPreviewControl_ShouldAddValidTextBox_IfBlockedElementsArePresent()
-        {
-            // Arrange
-            var svgPreviewControl = new SvgPreviewControl();
-
-            // Act
-            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));
-            var textBox = svgPreviewControl.Controls[0] as RichTextBox;
-
-            // Assert
-            Assert.IsFalse(string.IsNullOrWhiteSpace(textBox.Text));
-            Assert.AreEqual(textBox.Dock, DockStyle.Top);
-            Assert.AreEqual(textBox.BackColor, Color.LightYellow);
-            Assert.IsTrue(textBox.Multiline);
-            Assert.IsTrue(textBox.ReadOnly);
-            Assert.AreEqual(textBox.ScrollBars, RichTextBoxScrollBars.None);
-            Assert.AreEqual(textBox.BorderStyle, BorderStyle.None);
-        }

```

Comment:
Can you share what the user see's if the blocked elements are present now <script>?  Is there no way of detecting blocked elements now?

Reply:
With the current implementation the web browser will silently block the external content from loading and scripts to run. I have to figure out a way to show a pop up if something potentially blocked. With either just search for elements like ( `<image>, <script> `) in the Svg if present than show a pop up or having some kind of hook from web browser control if possible in case something is blocked. I'm thinking about addressing this in a separate PR.